routemaster/subscription.rb

```diff
@@ -0,0 +1,22 @@
+module Routemaster
+  class Subscription
```

Comment:
Sure thing. I may even namespace `Topic` under the same modules as well, it seems like it's only used in `monitor_topics`.